internal/scorecard/scorecard.go

```diff
@@ -107,28 +94,16 @@ func getPlugins(version string, selector labels.Selector) ([]Plugin, error) {
 	return plugins, nil
 }
 
-func Tests(cmd *cobra.Command, args []string) error {
-	if err := initConfig(); err != nil {
-		return err
-	}
-	if err := validateScorecardConfig(); err != nil {
-		return err
-	}
-
-	selector, err := labels.Parse(scViper.GetString(SelectorOpt))
-	if err != nil {
-		return err
-	}
-
-	cmd.SilenceUsage = true
-	plugins, err := getPlugins(scViper.GetString(schelpers.VersionOpt), selector)
-	if err != nil {
-		return err
+func (s Config) RunTests() error {
+	for idx, plugin := range s.PluginConfigs {
```

Comment:
Is this the only place that `PluginConfigs` is used in `RunTests()`? If so, I'm wondering if we can move the `validateConfig` function to be a member function of `PluginConfig`, and call it from the `cmd` package.

IIUC, that would mean that we would no longer need `PluginConfigs` to be in the `Config` struct.

Reply:
fixed in PR https://github.com/operator-framework/operator-sdk/pull/2428